includes/handler/class-delete.php

```diff
@@ -193,4 +179,18 @@ public static function defer_signature_verification( $defer, $request ) {
 
 		return false;
 	}
+
+	/**
+	 * Set the object to the object ID.
+	 *
+	 * @param \Activitypub\Activity\Activity $activity The Activity object.
+	 * @return \Activitypub\Activity\Activity The filtered Activity object.
+	 */
+	public static function outbox_activity( $activity ) {
+		if ( 'Delete' === $activity->get_type() ) {
+			$activity->set_object( $activity->get_object()->get_id() );
```

Comment:
we should ensure that it is an object before we try to get the ID, now that it is a filter!